### ISSUE TITLE
Toolbar ellipsis menu: improve menu labels contrast ratio and semantics

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#4123]
In the toolbar ellipsis menu, each menu has a visible label:

<img width="294" alt="screen shot 2017-12-21 at 14 02 07" src="https://user-images.githubusercontent.com/1682452/34257109-c0ef589c-e658-11e7-993b-05fb74957201.png">

1
The color contrast ratio of these labels is too low: they use a light gray `#a2aab2` against a white background and the contrast ratio is as low as 2.35:1. See https://jdlsn.com/color/?type=hex&color=a2aab2&color2=ffffff
In the current color palette, the lightest gray that can be used on a white background is `$dark-gray-300: #6c7781;`

2
These labels are just text within a `<div>` element. It would be great to use some more semantic markup, not sure if they should be headings though. Additionally, since the menus use `role="menu"`, it would be great to use these visible labels as target for `aria-labelledby`. This would require to use unique IDs on the labels to be used as reference in the `aria-labelledby` attributes on the menus.
